src/plugins/discover/public/application/components/discover_grid/get_render_cell_value.tsx

```diff
@@ -55,7 +55,7 @@ export const getRenderCellValueFn = (
   if (field && field.type === '_source') {
     if (isDetails) {
       // nicely formatted JSON for the expanded view
-      return <span>{JSON.stringify(row[columnId], null, 2)}</span>;
+      return <span>{JSON.stringify(row, null, 2)}</span>;
```

Comment:
it was a root cause of that issue, "_source" is the top level value and we need to return the entire row